client/src/index.css

```diff
@@ -5,9 +5,49 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  /* Styling for gradient background */
+  min-height: 90vh;
+  height: auto;
+  background: linear-gradient(#788be0, #6e3bdb);
+  background: -webkit-gradient(
+    linear,
+    left top,
+    left bottom,
+    from(#788be0),
+    to(#6e3bdb)
+  );
+  background: -o-linear-gradient(#788be0, #6e3bdb);
+  background: linear-gradient(#788be0, #6e3bdb);
+  background-repeat: no-repeat;
+  background-size: cover;
```

Comment:
This styling is not going to be on all pages - for that reason we should probably remove it from index.css and add it your useStyles hook 

In fact, it would be nice to have a resuable OnboardingContainer component that just contains all the styling for the background and the paper, and can be reused for Signup, Login, and the Onboarding Experience page that Daniyal will be working on

Reply:
OnboardingConatainer component created and used as a parent component which will recieve signup, login and experience as children.

index,css cleaned as suggested. Thanks